src/main/resources/io/github/dfdlschemas/envelopepayload/xsd/envelopePayload.dfdl.xsd

```diff
@@ -1,98 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<schema
-  targetNamespace="urn:io.github.dfdlschemas.envelopepayload"
-  xmlns:ep="urn:io.github.dfdlschemas.envelopepayload"
-
-  xmlns:msHdr="urn:milstd2045DFDL"
-  xmlns:pcap="urn:pcap:2.4"
-
-  xmlns="http://www.w3.org/2001/XMLSchema"
+<xs:schema
   xmlns:xs="http://www.w3.org/2001/XMLSchema"
-  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
   xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+  xmlns:tcpHdr="urn:io.github.dfdlschemas.tcpMessage"
+  xmlns:xsat2="urn:com.owlcyberdefense.xsat2"
```

Comment:
Can remove this xsat2 artifact. We really need a way to avoid having to modify the schema for xsat2 checking.

Reply:
We could do something like this sbt daffodil plugin does and define a key in the build.sbt that specifies the root file. It would require modifying the build.sbt but I think that's preferable over modifying the schema

Reply:
The problem with adding a key is that the key must be defined by the xsat2 plugin (or a dependency of the xsat2 plugin) so that it can use it. And if you don't have the xsat2 plugin installed (which most people don't since it's not publicly available) then SBT will error saying it doesn't know what the key is. I believe this schema is primarily an example so it's probably safe to remove xmlns:xsat2, and we can work on an alternative to the xsat2 plugin that doesn't require it in the meantime.